src/views/about.js

```diff
@@ -17,11 +17,11 @@ const HeLxIntro = styled.div`
   width: 80vw;
 `
 
-export const Home = () => {
+export const About = () => {
   return (
     <Container>
-      <Title>HeLx</Title>
-      <HomeContainer>
+      <Title>About HeLx</Title>
```

Comment:
Nit: May be worth asking Hannah if we want the title or just the text since they have navigated to the about page.

Reply:
I think we would get rid of this page and have a menu link to the future Helx marketing website from meeting this morning.